cfme/tests/automate/test_method.py

```diff
@@ -749,3 +750,50 @@ def test_null_coalescing_fields(request, klass):
     )
 
     assert log.validate()
+
+
+@pytest.mark.tier(3)
+def test_automate_user_has_groups(request, appliance, custom_instance):
+    """
+    https://bugzilla.redhat.com/show_bug.cgi?id=1411424
+    This method should work:  groups = $evm.vmdb(:user).first.miq_groups
+    $evm.log(:info, "Displaying the user"s groups: #{groups.inspect}")
+
+    Bugzilla:
+        1411424
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: medium
+        initialEstimate: 1/12h
+        tags: automate
+        startsin: 5.8
+    """
+    user, user_data = _users(request, appliance)
+
+    script = dedent(
+        f"""
+        group = $evm.vmdb(:user).find_by_name("{user[0].name}").miq_groups
+        $evm.log(:info, "Displaying the user's groups: #{{group.inspect}}")
+        """
+    )
+    instance = custom_instance(ruby_code=script)
+
+    with LogValidator(
+        "/var/www/miq/vmdb/log/automation.log",
+        matched_patterns=[f'.*{user_data[0]["group"]["description"]}.*'],
+    ).waiting(timeout=120):
+
+        # Executing automate method using simulation
+        simulate(
+            appliance=instance.klass.appliance,
+            message="create",
+            request="Call_Instance",
+            execute_methods=True,
+            attributes_values={
+                "namespace": instance.klass.namespace.name,
+                "class": instance.klass.name,
+                "instance": instance.name,
+            },
+        )
```

Comment:
**Suggestion:**
Here you can `assert` log for [this](https://github.com/ManageIQ/integration_tests/pull/9274/files#diff-b1e303070bc0d81681ca5ab5d5ad807aR778) string.

Reply:
No need to assert entire string.
If you run the test case then you will address that important thing is that, I need to check if I get the ```group``` of user associated to it by executing ```group.inspect```. Because here I am checking ```miq_groups``` with automate method.